cvpy/tests/test_cvat_project.py

```diff
@@ -288,30 +288,45 @@ def test_cvat_project_save_no_caslib_no_relative_path(self):
 
     def test_cvat_project_resume(self):
 
-        # Resume a test project
-        cvat_project = CVATProject.resume(project_name='MyDemoProject', cas_connection=self.cas_connection,
+        # Inner-class used for testing CVAT scenerios
+        def test_cvat_project_resume_assertions(cvat_project):
```

Comment:
We typically don't use inner-functions. I will suggest moving this outside as a separate non-inner function. You can call it something that starts with an underscore. e.g.

def _verify_cvat_project_attributes(cvat_project):